js_modules/dagit/packages/ui/src/components/TextInput.tsx

```diff
@@ -56,7 +56,7 @@ export const TextInputContainerStyles = css`
   position: relative;
 `;
 
-const Container = styled.div<{$disabled: boolean}>`
+export const TextInputContainer = styled.div<{$disabled: boolean}>`
```

Comment:
Also exporting this because I've found myself needing to customize the styling of this input (eg. to make the input 100% width)